ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -2564,9 +2574,11 @@ static void ggml_vk_print_gpu_info(size_t idx) {
         }
     }
 
+#if defined(GGML_VULKAN_COOPMAT_GLSLC_SUPPORT)
     if (!ggml_vk_khr_cooperative_matrix_support(props2.properties, driver_props)) {
         coopmat_support = false;
     }
+#endif
```

Comment:
This function is used twice, you only gated it once. But that's not needed, since it uses no logic from the extension, it's just a device info check. You can remove this #ifdef.